src/modules/vtol_att_control/vtol_att_control_main.cpp

```diff
@@ -531,6 +531,9 @@ void VtolAttitudeControl::task_main()
 			// update transition state if got any new data
 			if (got_new_data) {
 				_vtol_type->update_transition_state();
+
+				_vtol_type->process_mc_data();
+				fill_mc_att_rates_sp();
```

Comment:
@LorenzMeier which reminds me you mentioned a report of double publishing. Did you locate it already? I haven't looked yet but am happy to fix it if you already have some ideas where it is happening.